src/types.ts

```diff
@@ -203,3 +203,11 @@ export interface Organisation {
 }
 
 export interface UserOrg extends User, Organisation {}
+
```

Comment:
There's a types file in the "permissions" subfolder with other types used here already in it, so probably makes sense to keep this grouped with them (unless its used elsewhere?)

Reply:
Oh, ok - good point. I'll move to there and also add `displayName` in it